packages/@okta/vuepress-site/docs/reference/ssf-transmitter-sets/index.md

```diff
@@ -0,0 +1,301 @@
+---
+title: SSF Transmitter SET payload structures
+excerpt: Example JWT payload structures when an Okta event is fired.
+---
+
+# SSF Transmitter SET payload structures
+
+Okta uses the [Shared Signals Framework (SSF)](https://sharedsignals.guide/) to send security-related events and other data-subject signals to third-party security vendors.
+
+To enable the transmission of signals from Okta, you must create an [SSF stream](https://developer.okta.com/docs/api/openapi/okta-management/management/tag/SSFTransmitter/#tag/SSFTransmitter/operation/createSsfStream) using the [SSF Transmitter API](https://preview.redoc.ly/oktadev/jk-OKTA-912817/openapi/okta-management/management/tag/SSFTransmitter/). Then, configure the third-party receiver to accept signals from Okta.
```

Comment:
`SSF Transmitter API` should point to the production link: https://developer.okta.com/docs/api/openapi/okta-management/management/tag/SSFTransmitter/